### DISCUSSION TITLE
"Refactor" should use the "code-davinci-edit-001" model

### DISCUSSION THREAD

**Discussion by jakehemmerle** [JudiniLabs/code-gpt-docs#67] [Ideas]
Great extension! Thanks for making it. Really looking forward to the code being open sourced for privacy purposes and so contributing is easier.

A suggestion is that the new Codex editing model is used for refactoring code, 

https://platform.openai.com/docs/guides/code/editing-code

**Comment by davila7** [comment]
moved to discussion
